src/coreclr/tests/src/Common/Directory.Build.targets

```diff
@@ -21,17 +21,34 @@
       <RunTimeDependencyExcludeFiles Include="@(RunTimeDependencyExclude -> '%(FileName)%(Extension)')" />
       <RunTimeDependencyExcludeFiles Include="@(RunTimeDependencyExclude -> '%(FileName).ni%(Extension)')" />
       <RunTimeDependencyExcludeFiles Include="@(RunTimeDependencyExclude -> '%(FileName).pdb')" />
+
       <AllResolvedRuntimeDependencies Include="@(RuntimeCopyLocalItems -> '%(FileName)%(Extension)');@(NativeCopyLocalItems -> '%(FileName)%(Extension)')">
         <File>%(Identity)</File>
       </AllResolvedRuntimeDependencies>
+      
       <RunTimeDependencyCopyLocalFile Include="@(AllResolvedRuntimeDependencies)"  Exclude="@(RunTimeDependencyExcludeFiles)"/>
       <RunTimeDependencyCopyLocal Include="@(RunTimeDependencyCopyLocalFile -> '%(File)')"  />
       <RunTimeDependencyCopyLocal Include="$(TargetingPackPath)/*" />
+      
+      <RunTimeArtifactsExcludeFiles Include="protononjit.dll" />
+      <RunTimeArtifactsExcludeFiles Include="linuxonjit.dll" />
+      
+      <RunTimeArtifactsIncludeFolders Include="/" />
+      <RunTimeArtifactsIncludeFolders Include="crossgen2/" />
+      <RunTimeArtifactsIncludeFolders Include="PDB/" />
+      <RunTimeArtifactsIncludeFolders Include="R2RTest/" />
+
+      <!-- Add binary dependencies to copy-local items -->
+      <RunTimeDependencyCopyLocal
+          Include="$(CoreCLRArtifactsPath)/%(RunTimeArtifactsIncludeFolders.Identity)*"
```

Comment:
Small nit in these two lines: CoreCLRArtifactsPath should already contain a trailing `/`: https://github.com/dotnet/runtime/blob/4034228a3d3fa093e21479765f03c10c0443fc68/eng/liveBuilds.targets#L4

Reply:
Thanks for pointing that out, fixed in 2nd commit.